components/automate-chef-io/content/docs/applications.md

```diff
@@ -0,0 +1,145 @@
++++
+title = "Applications Tab"
+description = "Monitor Habitat Services in Chef Automate"
+date = 2019-10-18T18:54:09+00:00
+draft = true
+bref = ""
+toc = true
+[menu]
+  [menu.docs]
+    parent = "applications"
+    weight = 30
++++
+
+<!-- Task
```

Comment:
@mjingle We're almost there! I need your help revising this content. The "task" section that I've added & commented out is an information design/pattern for describing how to get things done.
Could you try it out on documenting the actions that a user can take on the Applications tab?
In particular, I think we need the context of what the Applications Tab does, and the steps, actions ("commands") and results that users can expect.